### PR TITLE
Added a warning for low ethBalance

### LINKED ISSUE #840: add a warning for check eth balance that warns user that balance is low

### PR DESCRIPTION
**Pull request by Yashk767**
# Description

Added a warning if ethBalance < 0.1
Fixes #840

### CODE REVIEW

File: cmd/vote.go [hunk 1]
```diff
@@ -165,6 +165,12 @@ func (*UtilsStruct) HandleBlock(client *ethclient.Client, account types.Account,
 		log.Errorf("Error in fetching balance of the account: %s\n%s", account.Address, err)
 		return
 	}
+
+	// Warning the staker if ETH balance is less than 0.1 ETH
+	if ethBalance.Cmp(big.NewInt(1e17)) == -1 {
+		log.Warn("Eth balance is lower than 0.1 ETH, kindly add more ETH to be safe for executing transactions successfully")
```
Comment (ashish10677) [inline cmd/vote.go:171]: This should be SKL or SFuel balance, not Eth

**Review by ashish10677 — CHANGES_REQUESTED** [review]

File: cmd/vote.go (hunk 1 above)
Comment (Yashk767) [reply cmd/vote.go:171]: okay. I will replace `ETH` with `SKL`

Comment (ashish10677) [reply cmd/vote.go:171]: Replace it with sFUEL.

Comment (ashish10677) [reply cmd/vote.go]: @Yashk767 replace it with sFUEL

**Pull request merged**